src/test/vstart_wrapper.sh

```diff
@@ -16,6 +16,7 @@
 # GNU Library Public License for more details.
 #
 
+CEPH_ROOT=${CEPH_ROOT:-..}
```

Comment:
@wjwithagen wouldn't we want "../.." to be the default relative path instead of ".." in order to reach the qa directory?